sdk/storage/faq.md

```diff
@@ -0,0 +1,374 @@
+  {
+    request.SetHeader("x-ms-custom-hearder-name", "header-value");
+    return nextPolicy.Send(request, context);
+  }
+};
+
+BlobClientOptions options;
+options.PerOperationPolicies.push_back(std::make_unique<NewPolicy>());
+options.PerRetryPolicies.push_back(std::make_unique<NewPolicy>());
+```
+
+## Will the SDK retry failed requests? What is the default retry logic? How can I customize retry behavior?
+
+Requests failed due to network errors or HTTP status code 408, 500, 502, 503, 504 will be retried at most 3 times (4 attempts in total) using exponential backup with jitter.
+These parameters can be customized with `RetryOptions`. Below is an example.
+
+```C++
+BlobClientOptions options;
+options.Retry.RetryDelay = std::chrono::milliseconds(800);
+options.Retry.MaxRetryDelay = std::chrono::seconds(60);
+options.Retry.MaxRetries = 3;
+```
+
+## Why do I see 503 Server Busy errors? What should I do in this case?
+
+Azure storage service has scalability and performance target, which varies for different types of accounts.
+When your application accesses storage too aggressively and reaches the limit, storage starts returning 503 Server Busy errors.
+You may refer to [this page](https://learn.microsoft.com/azure/storage/common/scalability-targets-standard-account) for the limits for different types of storage accounts in different regions.
+
+Here are a few things that turn out to be effective in practice to minimize the impact of this kind of error.
+
+1. Use exponential backoff policy and jitter for retries.
+   Exponential backoff allows the load on storage service to decrease over the time. Jitter helps ease out spikes and ensures the retry traffic is well-distributed over a time window.
+1. Increase retry count.
+1. Identify the throttling type and reduce the traffic sent from client side.
+   You can check the exception thrown from storage function calls with below code. The error message will indicate which scalability target was exceeded.
+   ```C++
+   try
+   {
+     // storage function goes here
+   }
+   catch (Azure::Storage::StorageException& e)
+   {
+     if (e.StatusCode == Azure::Core::Http::HttpStatusCode::ServiceUnavailable)
+     {
+       std::cout << e.Message << std::endl;
+     }
+   }
+   ```
+1. Store your data in multiple storage accounts for load balancing.
+1. Use CDN for static resources.
+
+Throttling error could also happen at subscription level or tenant level. You can still try above approaches or contact Azure Support in such cases.
+
+## How to troubleshoot 403 errors?
+
+403 error means your request to access Azure Storage is not correctly authorized.
+
+If you're using shared key authentication, you should
+
+1. Check the storage account name and account key match the resource you're trying to access, and the key is not rotated.
+1. If you need to add extra HTTP request headers or overwrite existing headers, do it with a HTTP pipeline policy, so that the new values can be signed with shared key.
```

Comment:
```suggestion
1. If you need to add extra HTTP request headers or overwrite existing headers, do it with an HTTP pipeline policy, so that the new values can be signed with the shared key.
```